features/admin.groups.v1/components/wizard/group-assign-users.tsx

```diff
@@ -270,86 +212,79 @@ export const AddGroupUsers: FunctionComponent<AddGroupUserProps> = (props: AddGr
-                                    usersList?.map((user: UserBasicInterface, index: number) => {
-                                        const header: string = user?.userName;
-                                        const subHeader: string = UserManagementUtils.resolveUserListSubheader(user);
-
-                                        return (
-                                            <TransferListItem
-                                                handleItemChange={ () => handleAssignedItemCheckboxChange(user) }
-                                                key={ index }
-                                                listItem={ {
-                                                    listItemElement: resolveListItemElement(header),
-                                                    listItemValue: header
-                                                } }
-                                                listItemId={ user.id }
-                                                listItemIndex={ index }
-                                                isItemChecked={
-                                                    checkedAssignedListItems?.filter((item: UserBasicInterface) =>
-                                                        item?.id === user?.id).length > 0
-                                                }
-                                                showSecondaryActions={ false }
-                                                showListSubItem={ true }
-                                                listSubItem={ subHeader && header !== subHeader && (
-                                                    <Header as="h6">
-                                                        <Header.Content>
-                                                            <Header.Subheader
-                                                                data-componentid={ `${ componentId }-item-sub-heading` }
-                                                            >
-                                                                { subHeader }
-                                                            </Header.Subheader>
-                                                        </Header.Content>
-                                                    </Header>
-                                                ) }
-                                                data-componentid={
-                                                    `${ componentId }-unselected-transfer-list-item-${ index }`
-                                                }
-                                            />
-                                        );
-                                    } )
-                                }
-                            </TransferList>
-                        </TransferComponent>
-                    </>
-                ) }
+            <Heading as="h5" className="mt-3">Add Users</Heading>
```

Comment:
shall we i18n the string literals?

Reply:
Fixed with 2de854728dc64722361ea106362f0f4c34c738b8